docs/errorhandling.rst

```diff
@@ -28,6 +28,54 @@ exception to the :attr:`~flask.Flask.logger`.
 But there is more you can do, and we will cover some better setups to deal
 with errors.
 
+.. _error-handlers:
+
+Error handlers
+--------------
+
+.. versionadded:: 1.0
+
+You might want to show error pages to the user when an error occurs.
+This can be done by registering error handlers.
+
+Error handlers are normal :ref:`views` but instead of being registered for
+routes they are registered exceptions that are rised while trying to do
+something else.
+
+Registering
+```````````
+
+Register error handlers using :meth:`~flask.Flask.errorhandler` or
```

Comment:
they do, i tested. the whole thing is rendered in python context apparently, and i see plain `:meth:` and `:exc:` all over the place